### ISSUE TITLE
lucky --version should also print the included framework versions

### ISSUE THREAD

**Issue by edwardloveall** [luckyframework/lucky_cli#99]
Instead of just printing 0.7.1 (or whatever the current version is), `lucky --version` should print out the versions each included framework it's going to create with `lucky init`

```bash
$ lucky --version
lucky_cli: 0.7.1
lucky: 0.6.2
lucky_migrator: 0.3.0
```

**Comment by edwardloveall** [comment]
I wonder what people expect this to print out. It seems obvious when not inside of a project, but would people expect it to show the versions currently installed in the shard.yml file, or the versions the executable will create?

**Comment by edwardloveall** [comment]
RSpec [does this](https://github.com/rspec/rspec-core/blob/048a8c7e9703f62c23a22f56f27b0fee9f53b543/lib/rspec/core/invocations.rb#L59-L69) well:

```bash
$ rspec -v
RSpec 3.7
  - rspec-core 3.7.0
  - rspec-expectations 3.7.0
  - rspec-mocks 3.7.0
  - rspec-rails 3.7.2
  - rspec-support 3.7.0
```

**Comment by jwoertink** [comment]
@edwardloveall Do you still see this as relevant? I was thinking about this, and feel like this could be confusing. For me, I just need to know the Lucky version I'm on, and then assume that all other versions (Dexter, Carbon, Avram, Wordsmith, Flow, Authentic, Habitat, Router) would just be the latest that is supported with that version of Lucky. Plus, that's a lot of shards we'd show 😅

**Comment by edwardloveall** [comment]
Hmmm... maybe. I could see it being useful but I don't think it needs to be `-v`. I could easily see it coming from a separate option like `--framework-versions`. The reason I like having something like this is it helps with support.

> User: I'm having an issue with X.
> Me: Can you run `lucky --framework-versions` and paste the result here?

I don't feel like it's critical though. We have the shards.yml file.

**Comment by jwoertink** [comment]
Cool. I like that.

**Comment by paulcsmith** [comment]
Could we get the same (or similar) info by  asking for the `shard.lock`? I imagine that's what `--framework-versions` would use under the hood.

**Comment by jwoertink** [comment]
I don't think so because in some cases it would show up with a commit hash instead of version number.

**Comment by paulcsmith** [comment]
That seems like it'd be really good to know. Since if they're using a specific commit it may not be the officially released version. But I may be thinking about this wrong

**Comment by jwoertink** [comment]
Ah, I see what you're saying. I would be good with either way.
